README.md

```diff
@@ -74,7 +87,7 @@ We have the following branches
  * **gh-pages**
    This contains the autogenerated code of the master branch that is generated by Travis.
  * **generator**
-   This branch is responsible for having the changes which will be implemented for generating the iso using webapp [meilix-generator](https://github.com/fossasia/meilix-generator)
+   This branch is responsible for having the changes which will be implemented for generating the iso using webapp [meilix-generator](https://github.com/fossasia/meilix-generator). It basically fetch the latest release from Github and use [mksquashfs tool](https://github.com/fossasia/meilix/blob/generator/build1.sh) to extract, made changes and then repack it and mail it. This will take very less time to customize the ISO.
 
 ## License
```

Comment:
https://github.com/fossasia/meilix/blob/master/LICENSE.md
 a link to file can also be added